### PR TITLE
Jimmy Granadino

### PR DESCRIPTION
**Pull request by jimmygranadino**
Vue.js + Socket.io chat app MVP with MongoDB + username/auth stretch goal (seems more likely than not or postgres for DB)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,88 @@
...
-When trying to scope your P4 think to how much you were able to accomplish during the hackathon, and keep your ideas to projects which include tasks that could fit that scope, plus some added features. 
+## Intended User
+Bored? I don't have a userbase but you can shout into the void until a userbase coalesces -- worked for the solar system on a long enough timeframe.
 
-*What does a non-passing P4 look like?*
-- If you do not make a meaninful attempt at trying and implementing a new technology
-- If you do not build anything substantial
-- If you do not put in meaningful work/git commits all six days of the project (including the weekend)
+## Team
+Myself as Dev and my cat, Paris, for moral support.
```
Comment (azocher) [inline README.md:10]: 😄

**Comment by azocher** [comment]
@jimmygranadino - this looks great! Overall I think it is an ambitious project, but excited to see even a version that gets close to this goal, even if it doesn't reach it 100%. Tagging @MilcahHalili @009Kings @nickubed for any additional feedback.

File: README.md (hunk 1 above)
Comment (jimmygranadino) [reply README.md:10]: lol she's truly a crucial team member

**Comment by MilcahHalili** [comment]
I think it's wise that you put auth and chat history as a stretch goal. Let me know if you need help with Socket.

**Comment by 009Kings** [comment]
Looks too to me with Milcah's scoping.

**Comment by nickubed** [comment]
Looks great to me!

**Pull request closed without merging**